spec/edib/defaults_spec.exs

```diff
@@ -19,6 +19,18 @@ defmodule EDIBDefaultsSpec do
       end
     end
 
+    describe "environment/0" do
+      it "returns prod as default environment" do
+        System.delete_env("MIX_ENV")
```

Comment:
We probably should mock the call to `System.get_env/1` instead, so to avoid possible conflicts if running the specs concurrently.

Something like the following should do the job:

```elixir
before do
  allow(SomeModule) |> to(accept :get_env, fn "MIX_ENV" -> nil end) # for delete
  allow(SomeModule) |> to(accept :get_env, fn "MIX_ENV" -> "staging" end) # for different values
end
```

Could you try this and adjust the PR if it's working?